package.json

```diff
@@ -22,7 +22,7 @@
     "rewrite"
   ],
   "dependencies": {
-    "detective": "~4.1.0",
+    "detective": "4.x.x",
```

Comment:
Done. In the end I added ~ as well to make sure that npm uses the latest version in case there are other packages with the same dependency